Source/relay/TourGuide/Paths/Avatar of Shadows over Loathing.ash

```diff
@@ -23,3 +23,63 @@ void PathShadowsOverLoathingGenerateTasks(ChecklistEntry [int] task_entries, Che
         task_entries.listAppend(ChecklistEntryMake("__skill Motif", "", ChecklistSubentryMake(main_title, "", description), -11));
     }
 }
+
+record CursedItem {
+    item theItem;
+    monster boss;
+    string description;
+    boolean shouldDisplay;
+};
+
+RegisterResourceGenerationFunction("PathShadowsOverLoathingGenerateResource");
+void PathShadowsOverLoathingGenerateResource(ChecklistEntry [int] resource_entries) {
+    if (my_path() != $path[Avatar of Shadows Over Loathing]) return;
+
+    CursedItem [int] cursedItems = {
+        new CursedItem(
+            $item[cursed bat paw],
+            $monster[two-headed shadow bat],
+            "+25 ML",
+            __quest_state["Boss Bat"].finished != true
```

Comment:
Could trigger each of these based on having the cursed item itself instead too – let me know if you all have opinions.

Reply:
nah, i think this is better; otherwise, any that the user uncurses would add the item back to the list, which is unnecessary and probably bad